webapi/src/main/scala/org/knora/webapi/responders/v2/StandoffResponderV2.scala

```diff
@@ -93,73 +101,58 @@ class StandoffResponderV2(
     case GetXSLTransformationRequestV2(xsltTextReprIri, requestingUser) =>
       getXSLTransformation(xsltTextReprIri, requestingUser)
-    case other => handleUnexpectedMessage(other, log, this.getClass.getName)
+    case other => Responder.handleUnexpectedMessage(other, this.getClass.getName)
   }
 
-  private val xsltCacheName = "xsltCache"
-
-  private def getStandoffV2(getStandoffRequestV2: GetStandoffPageRequestV2): Future[GetStandoffResponseV2] = {
-    val requestMaxStartIndex = getStandoffRequestV2.offset + responderData.appConfig.standoffPerPage - 1
+  private def getStandoffV2(getStandoffRequestV2: GetStandoffPageRequestV2): Task[GetStandoffResponseV2] = {
+    val requestMaxStartIndex = getStandoffRequestV2.offset + appConfig.standoffPerPage - 1
 
     for {
-      resourceRequestSparql <- Future(
-                                 org.knora.webapi.messages.twirl.queries.sparql.v2.txt
-                                   .getResourcePropertiesAndValues(
-                                     resourceIris = Seq(getStandoffRequestV2.resourceIri),
-                                     preview = false,
-                                     withDeleted = false,
-                                     maybePropertyIri = None,
-                                     maybeVersionDate = None,
-                                     queryAllNonStandoff = false,
-                                     maybeValueIri = Some(getStandoffRequestV2.valueIri),
-                                     maybeStandoffMinStartIndex = Some(getStandoffRequestV2.offset),
-                                     maybeStandoffMaxStartIndex = Some(requestMaxStartIndex),
-                                     stringFormatter = stringFormatter
-                                   )
-                                   .toString()
-                               )
-
-      // _ = println("=================================")
-      // _ = println(resourceRequestSparql)
-
-      // standoffPageStartTime = System.currentTimeMillis()
+      resourceRequestSparql <-
+        ZIO.attempt(
+          org.knora.webapi.messages.twirl.queries.sparql.v2.txt
+            .getResourcePropertiesAndValues(
+              resourceIris = Seq(getStandoffRequestV2.resourceIri),
+              preview = false,
+              withDeleted = false,
+              maybePropertyIri = None,
+              maybeVersionDate = None,
+              queryAllNonStandoff = false,
+              maybeValueIri = Some(getStandoffRequestV2.valueIri),
+              maybeStandoffMinStartIndex = Some(getStandoffRequestV2.offset),
+              maybeStandoffMaxStartIndex = Some(requestMaxStartIndex),
+              stringFormatter = stringFormatter
+            )
+            .toString()
+        )
 
-      resourceRequestResponse: SparqlExtendedConstructResponse <-
-        appActor
-          .ask(
+      resourceRequestResponse <-
+        messageRelay
+          .ask[SparqlExtendedConstructResponse](
```

Comment:
Use the `TriplestoreService` here. Simply use constructor injection to get an instance.

Please do this for all other places where a query is necessary.

Reply:
Done in [0b63e5b](https://github.com/dasch-swiss/dsp-api/pull/2498/commits/0b63e5b13df0fd9c33d47ee6ca547b4de2102cf1).